<ui version="4.0" >
 <class>GelmanRubin</class>
 <widget class="QDialog" name="GelmanRubin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Gelman-Rubin test</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="2" column="0" >
      <widget class="QPushButton" name="exportButton" >
       <property name="text" >
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QTableWidget" name="table" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="rowCount" >
        <number>5</number>
       </property>
       <property name="columnCount" >
        <number>4</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Result of the Gelman-Rubin test:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="baseSize" >
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title" >
      <string>Test options:</string>
     </property>
     <widget class="QRadioButton" name="radioType1" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string>Parameters only (fastest)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioType2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string>Parameters + Edges (for constant trees)</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioType3" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>60</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string>Parameters + Pairwise Distances (for varying trees)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioType4" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>80</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string>Parameters + Edges + Pairwise Distance (for fixed topology)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioType5" >
	     <property name="geometry" >
		     <rect>
			     <x>10</x>
			     <y>100</y>
			     <width>391</width>
			     <height>22</height>
		     </rect>
	     </property>
	     <property name="text" >
		     <string>Branch lengths only</string>
	     </property>
     </widget>
     <widget class="QPushButton" name="pushGo" >
      <property name="geometry" >
       <rect>
        <x>470</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text" >
       <string>Perform Test</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
